src/components/form/fields/simpleField.jsx

```diff
@@ -12,25 +12,26 @@ const SimpleField = props => {
         onChange,
         showLabel,
         inputId,
-        type,
-        required,
         label,
-        autoComplete,
         placeholder = label
     } = props;
+
     return (
-        <FormGroup inputId={inputId}
+        <FormGroup
+            inputId={inputId}
             labelText={label}
             {...pick(validation, 'error')}
             showLabel={showLabel}>
-            <Input id={inputId}
+            <Input
+                id={inputId}
                 name={path}
-                type={type}
+                type={props.type}
                 value={value || ''}
                 placeholder={placeholder}
-                autoComplete={autoComplete}
+                autoComplete={props.autoComplete}
                 title={label}
-                required={required}
+                required={props.required}
+                readOnly={props.readOnly}
```

Comment:
Why not use the destructuring like above?

Reply:
If the variable is not used twice, I don't see the point of creating a variable for it and we can see directly from where the property comes (in this example, the props).